Example/Tests/UIButtonBuilderTests.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  UIButtonBuilderTest.swift
+//  MobilliumBuilders_Tests
+//
+//  Created by Murat Celebi on 17.03.2021.
+//  Copyright © 2021 CocoaPods. All rights reserved.
```

Comment:
We can change copyright to Mobillium
```suggestion
//  Copyright © 2021 Mobillium. All rights reserved.
```